packages/shared/components/organisms/AccountActivity.svelte

```diff
@@ -53,6 +52,16 @@
         }
     })
 
+    $: $activityFilter, $activitySearchTerm, scrollToTop()
+    $: isEmptyBecauseOfFilter =
+        $selectedAccountActivities.filter((_activity) => !_activity.isHidden).length > 0 &&
+        activityListWithTitles.length === 0
+
+    function scrollToTop() {
+        const listElement = document.querySelector('.activity-list')?.querySelector('svelte-virtual-list-viewport')
+        if (listElement) listElement.scroll(0, 0)
```

Comment:
Let's keep if statements more explicit.
```suggestion
        if (listElement) {
            listElement.scroll(0, 0)
        }
```